packages/elpi/elpi.1.10.0/opam

```diff
@@ -10,7 +10,7 @@ bug-reports: "https://github.com/LPCIC/elpi/issues"
 build: [
   ["dune" "subst"] {pinned}
   ["dune" "build" "-p" name "-j" jobs]
-  [make "tests" "DUNE_OPTS=-p %{name}%" "TIMEOUT=120"] {with-test & os != "macos"}
+  [make "tests" "DUNE_OPTS=-p %{name}%" "TIMEOUT=3600"] {with-test & os != "macos"}
```

Comment:
This is an hour's timeout -- that's quite long for a CI test. Does it take that long?

Reply:
I'm not sure how long it can take. It all depends on the server I guess. 2 minutes was not enough so I though an hour sound be good in all cases (our CI systems have their own timeouts anyway)

I just lowered it to 15mins if that sounds better.